requirements.txt

```diff
@@ -1,6 +1,6 @@
-    pytket[ZX] == 1.17.1
+    pytket[ZX] == 1.18.0
     pytket-projectq == 0.28.0
-    pytket-qiskit == 0.41.0
+    pytket-qiskit == 0.42.0
     pytket-qsharp == 0.34.0
     pytket-cirq == 0.29.0
-    pytket-pyquil == 0.29.0
+    pytket-pyquil == 0.29.0
```

Comment:
Did you delete a newline or something? That may explain why it can't find `pytket.extensions.pyquil`.

Reply:
I think it is missing a new line, the fail was showing up because I had removed this in the first run to debug my not working build on the ci, but it was fine here so I added it back in and it still worked here

Reply:
( see [ae9e391](https://github.com/CQCL/pytket/pull/247/commits/ae9e39179218f93f3bcea52170a06eb4b222d71c) )